build.gradle

```diff
@@ -191,29 +191,30 @@ dependencies {
     implementation 'com.google.cloud:google-cloud-storage'
     implementation 'com.google.cloud:google-cloud-firestore'
     implementation 'com.google.cloud:google-cloud-pubsub'
+    implementation 'com.google.cloud:spring-cloud-gcp-starter-logging:4.9.0'
     implementation 'com.google.http-client:google-http-client'
     implementation 'org.apache.commons:commons-dbcp2:2.7.0'     // For database connection support
     implementation 'org.apache.commons:commons-lang3:3.11'
     implementation 'org.apache.commons:commons-collections4:4.4'
     implementation 'org.apache.directory.studio:org.apache.commons.io:2.4'
     implementation 'org.liquibase:liquibase-core'
     implementation 'org.postgresql:postgresql'          // Postgres jdbc driver
-    implementation 'ch.qos.logback:logback-core:1.2.13'      // Unpin logback versions with DR-3404 Spring 3 Upgrade
-    implementation 'ch.qos.logback:logback-classic:1.2.13'  // logback-classic natively implements slf4j api
+    implementation 'ch.qos.logback:logback-core'
+    implementation 'ch.qos.logback:logback-classic'  // logback-classic natively implements slf4j api
 
-    implementation 'org.codehaus.janino:janino:3.1.11'           // Provides if/else xml parsing for logback config
+    implementation 'org.codehaus.janino:janino'           // Provides if/else xml parsing for logback config
     implementation 'org.springframework.boot:spring-boot-starter-web'
-    implementation 'javax.validation:validation-api'
+    implementation 'jakarta.validation:jakarta.validation-api'
+    implementation 'jakarta.annotation:jakarta.annotation-api'
+
+    implementation 'org.apache.httpcomponents.client5:httpclient5'
```

Comment:
There is a note about this in the description:

> Apache HttpClient in RestTemplate
Support for Apache HttpClient has been https://github.com/spring-projects/spring-framework/issues/28925, immediately replaced by org.apache.httpcomponents.client5:httpclient5 (note: this dependency has a different groupId). If you are noticing issues with HTTP client behavior, it could be that RestTemplate is falling back to the JDK client. org.apache.httpcomponents:httpclient can be brought transitively by other dependencies, so your application might rely on this dependency without declaring it.